CMakeModules/BuildDepsInline.cmake

```diff
@@ -0,0 +1,97 @@
+
+include(ProcessorCount)
+ProcessorCount(NUM_CORES)
+
+set(BOOST_ROOT "${CMAKE_CURRENT_BINARY_DIR}/boost_install")
+set(MONGO_DRIVER_ROOT "${CMAKE_CURRENT_BINARY_DIR}/mongo_driver_install")
+set(WASM_ROOT "${CMAKE_CURRENT_BINARY_DIR}/wasm_install")
```

Comment:
WASM_ROOT is a misnomer -- it's just a build of llvm4 being placed in here

Reply:
Done